Quickfacts update revised 2/1/2019

lh1008 opened the pull request:
Hi @erciccione,

Just added new updates for the Quick-Facts Sheet, this includes reorganization of some text inside the .md and the .sla file. Stats were added revised by February and available for the community once is merged into the repo.

Thank you.

Will be opening an issue calling translators to update the .md file.

lh1008 commented:
Had to fix the revised date from the last commit. Forgot to change 2018 to 2019. @v1docq47 noticed the date error.

Thank you @v1docq47

el00ruobuob commented on monero-outreach-docs/en/quickfacts/quickfacts.md:
I seems to be more 0.65% at that time

lh1008 replied:
This were data taken from the date 2/01/2019, maybe today is already at 0.65%. If we take a look at all the data today everything would change. We are actually doing it every 3 or 4 months.

lh1008 replied:
Maybe we should start on updating the file in shorter periods of time. What do you think @el00ruobuob?

el00ruobuob replied:
My point was that 0.77 has not been updated, and looking at February the 1st on coinmarkcap, i am closer to 0.65 than 0.77

lh1008 replied:
You are right, I took the Monero marketcap from feb 01 and the global marketcap and gave me ~0.65. I used coinmarketcap as a source. 
Marketcap
Monero - feb01 732,932,071 
https://coinmarketcap.com/currencies/monero/historical-data/?start=20181205&end=20190305
Global historical data
https://coinmarketcap.com/charts/
Global - feb01 UTC -5:00 112,421,312,048

The good thing is that we are able to verify the historical data. Wasn't sure that was something we were able to do, but well, it has been quite a learning curve. Thank you @el00ruobuob will do another commit to add this changes.

lh1008 replied:
changed, done

el00ruobuob commented on monero-outreach-docs/en/quickfacts/quickfacts.md:
```diff
@@ -97,11 +98,9 @@ _With “tail emission” of 0.6 XMR/block, by 2040 there will be an equal amoun
 **Kovri:** This is a major privacy upgrade, adding a fourth layer of privacy to Monero transactions. Kovri uses **[garlic-encryption](https://getmonero.org/resources/moneropedia/garlic-encryption.html)** (think of Tor on steroids) to mask the IP addresses of senders and receivers using the network. Every layer of a Monero transaction, from the sender, receiver, transaction amount, and (with Kovri ...
 
-**Hardware Wallets:** Trezor Model T hardware wallet added Monero to their **[supported coins](https://trezor.io/coins/#XMR)**. **[Ledger hardware wallet](https://support.ledgerwallet.com/hc/en-us/articles/360006352934-Monero-XMR-Advanced)** already available through official Monero GUI wallet. The Monero community funded a team that is building **[Kastelo, an open source hardware wallet](http:// ...
+**Hardware Wallets:** The popular Ledger and Trezor hardware wallets are working on adding Monero. (Ledger is supported since CLI 0.12.2 and GUI 0.12.3) The Monero community is funding a team that is **[building a hardware wallet](http://kastelo.org/)** built on the Monero ethos. Both of these hardware wallet options are anticipated to be operational by the end of 2018.
```
`both of these`: but we are counting three.  
`anticipated at the end of 2018` but we are in 2019.

lh1008 replied:
Yeah I think it needs some reorganized structure, but it refers to Trezor and Ledger.

lh1008 replied:
Did the changed on this, let me know what you think :).

el00ruobuob replied:
I can't see the change atm. Did you pushed it?

lh1008 replied:
Yes in a different commit, let me squash so you are able to see the changes.

el00ruobuob commented on monero-outreach-docs/en/quickfacts/quickfacts.md:
```diff
@@ -1,12 +1,12 @@
+Users can spend safely, no one can see their balances or track their activity unless the user gives them permission.
 
-Revised 11/14/2018 - [Printer Friendly Version](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_PrinterFriendly.pdf)    
+Revised 2/1/2019 - [Printer Friendly Version](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_PrinterFriendly.pdf) 
```
shouldn't it link to a 2019 folder?

lh1008 replied:
I was also thinking about opening a new folder for 2019, I think on the next update we could add it so we don't mess everything right now, I've seen some PR already being submitted.

el00ruobuob replied:
ok

el00ruobuob commented on monero-outreach-docs/en/quickfacts/quickfacts.md:
```diff
@@ -1,12 +1,12 @@
+Revised 2/1/2019 - [Printer Friendly Version](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_PrinterFriendly.pdf) 
 Created for the community by Monero Outreach
 
 - [Full Color Version](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts.pdf)
```
I think this should be a 2019 folder too.

lh1008 replied:
Same as last comment, we might do it for the next update so we won't mess everything right now.

el00ruobuob replied:
ok

el00ruobuob commented:
hello @lh1008, did you had a chance to look at the suggestions i made?

lh1008 commented:
Hey @el00ruobuob,

Yes, the data that is currently was the revised by the date 2/1/2019, so if we see the information by today we are a month behind. I believe you have been looking at the currently .md file inside the repo. I think @erciccione hasn't merged the last PR #99 with the update, in the issue #100 I forward to that PR. 

Let me see.

lh1008 commented on monero-outreach-docs/en/quickfacts/quickfacts.md:
```diff
@@ -26,19 +26,20 @@ As a decentralized cryptocurrency, Monero is secured by a large network of users
 #### SCALABILITY
 
-Monero’s _[dynamic block size](https://www.mycryptopedia.com/block-size-explained/)_ changes based on transaction volume to provide lower fees and faster transactions. Higher transaction volume leads to larger block size limit, whereas low volume leads to a smaller block size limit. _[Bulletproofs](https://web.stanford.edu/~buenz/pubs/bulletproofs.pdf)_ also helps scability by significantly reduc ...
+Unlike Bitcoin and other projects that often rely on hardcoded constants, Monero’s _[dynamic block size](https://www.mycryptopedia.com/block-size-explained/)_ adapts automatically to thevolume of transactions, providing lower fees and faster confirmations.
```
done :)

lh1008 commented:
hey @el00ruobuob,

Did some minor changes, let me  know if this are good. And I'm waiting for your comments regarding the 2019 folder. Or maybe we once merged I can submit the 2019 folder once we are done.

lh1008 commented:
Btw, guys, sorry if I deleted part of my first comment, I thought it was on another PR and got confused, so I deleted my message and then made the suggested changes and added the commit.

el00ruobuob commented:
@lh1008 no problem with the folder, i just wonder of it could be automatically pushed in a 2019 folder (something Jekyll will did for instance). I am fine with any folder ;)

lh1008 commented:
Ok guys, updated the files and the .md format also. 

Let me know if everything is ok.

@el00ruobuob @erciccione

erciccione commented on monero-outreach-docs/en/quickfacts/quickfacts.md:
```diff
@@ -1,12 +1,12 @@
 # MONERO
 
-Monero is a secure, private, untraceable digital currency that is available to all.
+Monero is a secure, private, untraceable digital currency that is available to all. It’s features make it fungible cash.
```
*its* features

lh1008 replied:
wow, I have been missing some strange things. Thank you erciccione. Done.

erciccione commented on monero-outreach-docs/en/quickfacts/quickfacts.md:
```diff
@@ -51,42 +52,42 @@ Monero has made several large improvements since launch. Nearly all improvements
-**o:** a thing (grammatically speaking: a noun)   
+**mon:** money
+**er:** the smallest part
+**o:** a thing (grammatically speaking: a noun)
```
This block will show up on only one line if the trailing spaces at the end are removed (4 were too many btw)

lh1008 replied:
Had to leave spaces between sentences because adding spaces at the end didn't work. Now I think this should be ready to merge.

lh1008 commented:
Did the final edits to the .md and the rest of the files. This should be ready to merge unless there is something else missing. Thank you guys for your support and work.
